aws-geo-location/src/androidTest/java/com/amplifyframework/geo/location/MapsApiTest.kt

```diff
@@ -92,12 +91,20 @@ class MapsApiTest {
     }
 
     private fun signInWithCognito() {
-        val context = ApplicationProvider.getApplicationContext<Context>()
-        val (username, password) = Credentials.load(context)
-        auth?.signIn(username, password)
+        try {
```

Comment:
do you we know why is signing failing? we might mask some other issue by just swallowing the error.

Reply:
It fails randomly due to late response from the sign in api. This is just going to make sure the GEO tests do not break due to this API.